packages/contracts/contracts/L1/L2OutputOracle.sol

```diff
@@ -1,25 +1,33 @@
 //SPDX-License-Identifier: MIT
 pragma solidity 0.8.10;
 
-/* Library Imports */
 import { Ownable } from "@openzeppelin/contracts/access/Ownable.sol";
 
 /**
  * @title L2OutputOracle
- * @notice
+ * @notice The L2 state is committed to in this contract
+ * The payable keyword is used on appendL2Output to save gas on the msg.value check.
+ * This contract should be deployed behind an upgradable proxy
  */
-// The payable keyword is used on appendL2Output to save gas on the msg.value check.
 // slither-disable-next-line locked-ether
 contract L2OutputOracle is Ownable {
     /**********
      * Events *
      **********/
 
     /// @notice Emitted when an output is appended.
-    event l2OutputAppended(bytes32 indexed _l2Output, uint256 indexed _l2timestamp);
+    event l2OutputAppended(
+        bytes32 indexed _l2Output,
+        uint256 indexed _l1Timestamp,
+        uint256 indexed _l2timestamp
```

Comment:
Should also use block number here. This is a larger refactor though.

Reply:
We need consensus before making that large of a change, should put together a doc @trianglesphere  and schedule discussion for next office hours @K-Ho